libsolidity/formal/CHC.cpp

```diff
@@ -16,6 +16,7 @@
 */
 // SPDX-License-Identifier: GPL-3.0
 
+#include "libsmtutil/SolverInterface.h"
```

Comment:
Is it complaining for this?
```suggestion
#include <libsmtutil/SolverInterface.h>
```

Reply:
Yep